q2_types/feature_table/_type.py

```diff
@@ -30,5 +30,10 @@
 
 plugin.register_semantic_type_to_format(
```

Comment:
Yup, this wouldn't remove support for v1, it just makes sure v2.1 is the default.